.eslintrc

```diff
@@ -3,7 +3,6 @@
   "parser": "babel-eslint",
   "rules": {
     "arrow-parens": [2, "as-needed"],
-    "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.test.js", "**/*.spec.js"]}],
```

Comment:
is this needed here? Because we maintain this file in tools too!

Reply:
Oh on my vscode it completly break eslint :/ I can re-add it but it just fails to start eslint server due to that

Reply:
May I ask you to do it in another PR? (tools first then go back here to fix it)